templates/cppLibrary/android/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+set (CMAKE_VERBOSE_MAKEFILE ON)
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
Maybe 14 ? 11 is bit of old.

Reply:
I had issues with newer versions as it's compiled for ios and android. I will try to investigate it more in future to bump it.